### DISCUSSION TITLE
Voice Isolation - popping

### DISCUSSION THREAD

**Discussion by TomThistlethwaite** [CommandPost/FCPCafe#266] [Final Cut Pro Questions] [answered]
Hey there, 

Just wondering if anyone else has found that audio popping can occur on cuts when using the Voice Isolation feature - notably when crossfading audio clips (using handles). Annoyingly, there seems to be no rhyme or reason as to when pops, except that they disappear when the Voice Isolation is turned off. Also irritating is that they sometimes don't reproduce on an export and then sometimes do. 

Cheers,

Tom

**Comment by latenitefilms** [comment, marked as answer]
This has definitely been widely reported, especially if you're working with Multicam Clips or Sync Clips. I find you can generally tweak fade handles to make the clicks disappear.

**Reply by TomThistlethwaite** [reply]
Thanks Chris! Glad I'm not alone. Yep lots of adjusting the fade handles, but what's annoying is how random it is. My tweak at the moment seems to be only applying the voice isolation after making a compound clip for a final role audio mix, but that's less than ideal.
